pandas/tests/indexes/test_interval.py

```diff
@@ -681,6 +682,47 @@ def f():
 
         pytest.raises(ValueError, f)
 
+    def test_is_non_overlapping_monotonic(self):
+        # Verify that a Python Boolean is returned (GH17237)
+        for closed in ('left', 'right', 'neither', 'both'):
```

Comment:
this test is superfluous as

assert .... is True check for boolness already, e.g.

```
In [3]: type(np.array(1) > 0)
Out[3]: numpy.bool_

In [5]: (np.array(1) > 0) is True
Out[5]: False

In [7]: assert bool(np.array(1) > 0) is True

In [8]: assert (np.array(1) > 0) is True
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-8-6614f944ebb4> in <module>()
----> 1 assert (np.array(1) > 0) is True

AssertionError: 
```